Library/Formula/git-cmd-hub.rb

```diff
@@ -0,0 +1,25 @@
+class GitCmdHub < Formula
+  desc "Interact with GitHub's v3 REST API using the git hub subcommand"
+  homepage "https://github.com/ingydotnet/git-hub"
+  url "https://github.com/ingydotnet/git-hub/archive/0.1.5.tar.gz"
+  sha256 "c057c2c56fa84a389105f935f63a72e5fa3f9fb9412a273d87b403b525ee2225"
+
+  def install
+    inreplace "Makefile" do |f|
+      # Prevent the makefile from installing outside of the prefix
+      f.gsub! "$(shell git --exec-path | sed 's/.*://')", "$(PREFIX)/bin"
+    end
+
+    inreplace "lib/git-hub" do |f|
+      # Fix the hardcoded directory to match Homebrew's structure
+      f.gsub! "SELFDIR=\"$(cd -P `dirname $BASH_SOURCE` && pwd -P)\"", "SELFDIR=#{bin}"
+    end
+
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "[ -f #{File.join(man1, "git-hub.1")} ]"
+    system "[ -f #{File.join(bin, "git-hub")} ]"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!